Assets/XML/Text/Technologies.xml

```diff
@@ -170,24 +178,29 @@
 		<German>Es wird vermutet, dass Bronze, eine Legierung aus Kupfer und Zinn, bereits im Jahre 4500 vor Christus in Thailand verwendet wurde. Mehrere Jahrhunderte sp&#228;ter wurde die Bronzeverarbeitung im Nahen Osten entdeckt. Kunstwerke dieser Periode zeigen, dass Bronze anfangs nur zu Dekorationszwecken genutzt wurde, weil Zinn schwer zu beschaffen war. Erst um etwa 2000 vor Christus wurde Zinn regelm&#228;&#223;ig von den benachbarten L&#228;ndern importiert. In der Folgezeit wurde das bis dahin verwendete Kupfer bei der Herstellung gr&#246;&#223;erer Objekte durch Bronze ersetzt, die h&#228;rter und weniger spr&#246;de war als Kupfer, so dass sich damit sch&#228;rfere Kanten formen lie&#223;en - dar&#252;ber hinaus waren Werkzeuge, Waffen und R&#252;stungen aus Bronze auch preiswerter und langlebiger. Die Zeit, in der Bronze das bevorzugte Metall f&#252;r die Fertigung dieser Ger&#228;te war, ist als Bronzezeitalter bekannt. Diese Periode dauerte bis etwa 1200 vor Christus, als das noch hochwertigere Eisen verf&#252;gbar wurde.</German>
-		<Italian>Le prime testimonianze sull'uso del bronzo, una lega di rame e stagno, risalgono alla Thailandia del 4500 a. C. Diversi secoli pi� tardi, la lavorazione del bronzo venne scoperta nel Medio Oriente. Gli oggetti risalenti a questo periodo dimostrano che il bronzo veniva inizialmente utilizzato solo a scopo decorativo, questo perch� lo stagno non era facilmente reperibile. Bisogner� aspettare pi� o meno il 2000 a. C. perch�, con l'importazione regolare di stagno dalle regioni confinanti, il bronzo sostituisca il rame nella fabbricazione di oggetti pi� grandi, offrendo anche una maggior resistenza rispetto al rame e la possibilit� di produrre punte pi� acuminate. Strumenti, armi e armature di bronzo erano anche meno costosi da realizzare e pi� duraturi. Il periodo di tempo in cui si preferiva il bronzo per tali prodotti � noto come l'Et� del Bronzo, che dur� fino al 1200 a. C. circa, quando fu soppiantato dalla lavorazione del ferro.</Italian>
+		<Italian>Le prime testimonianze sull'uso del bronzo, una lega di rame e stagno, risalgono alla Thailandia del 4500 a. C. Diversi secoli pi� tardi, la lavorazione del bronzo venne scoperta nel Medio Oriente. Gli oggetti risalenti a questo periodo dimostrano che il bronzo veniva inizialmente utilizzato solo a scopo decorativo, questo perch� lo stagno non era facilmente reperibile. Bisogner� aspettare pi� o meno il 2000 a. C. perch�, con l'importazione regolare di stagno dalle regioni confinanti, il bronzo sostituisca il rame nella fabbricazione di oggetti pi� grandi, offrendo anche una maggior resistenza rispetto al rame e la possibilit� di produrre punte pi� acuminate. Strumenti, armi e armature di bronzo erano anche meno costosi da realizzare e pi� duraturi. Il periodo di tempo in cui si preferiva il bronzo per tali prodotti � noto come l'Et� del Bronzo, che dur� fino al 1200 a. C. circa, quando fu soppiantato dalla lavorazione del ferro.</Italian>
```

Comment:
Can you run this through the converter too? The original text is to blame, but your editor likely made the problem worse.

Reply:
Sure